schedule/yast/autoyast/autoyast_opensuse_rules_and_classes.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: autoyast_rules_and_classes
+description: >
+  For future testing of AutoYaST rules and classes
+vars:
+  AUTOYAST: autoyast_opensuse/rule-based_example/
+schedule:
+  - autoyast/prepare_rules_and_classes
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/clone
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/validate_partition_table_via_blkid
+  - console/validate_blockdevices
+  - console/zypper_lr
+  - console/zypper_ref
+  - console/ncurses
+  - update/zypper_up
+  - console/zypper_in
+  - console/zypper_log
+  - console/orphaned_packages_check
+  - autoyast/verify_cloned_profile
```

Comment:
please, exclude all test that are using test data (they are not doing anything silently), so basically:
```
...
- installation/first_boot
- console/zypper_lr
- console/zypper_ref
- console/ncurses
- update/zypper_up
- console/zypper_in
- console/zypper_log
- console/orphaned_packages_check
```
But if you remove everything after first_boot also is ok. Could you create a ticket with low priority to remind us to do proper validation with this test suites?

Reply:
probably you closed the conversation accidentally with the squashing.